File: rata-rata.c
#include <stdio.h>
//definisi prosedur
void rata_rata () {
    int jumlah = 0;
    int i, angka;
    float hasil;
    for (i=0; i<5; i++) {
        printf("Masukkan angka ke-%d: \n",i + 1);
        scanf("%d", &angka);
        jumlah = jumlah + angka;
    } 
    hasil = (float)jumlah/5;
    printf("rata-rata: %.2f", hasil);
}
//pemanggilan prosedur
int main ()
{
    rata_rata();

    return 0;
} 

File: rata-rata.py
# definisi prosedur
def rata_rata():
    jumlah = 0
    for i in range(5):
        angka_str= input(f"Angka ke-{i + 1}: ")
        angka = int(angka_str)
        jumlah += angka
    print(f"jumlah: {jumlah}")
    hasil = jumlah / 5
    print(f"rata-rata: {hasil}")
# pemanggilan prosedur
rata_rata()